packages/gatsby/cache-dir/static-entry.js

```diff
@@ -334,6 +345,17 @@ export default (pagePath, callback) => {
       />
     )
   }
+  if (appData) {
```

Comment:
Can it ever bu falsy for production builds? (for develop builds we use `develop-static-entry.js`)

Reply:
Technically not but I just copied the code from page data. I was thinking of going through this file in a later stage and rewrite it a bit.

We don't need this in develop :)

Reply:
For sure we don't need it for develop (was just mentioning this in case the check was to not include it in develop or something)